src/components/dialogs/parameters/dynamicsimulation/dynamic-simulation-parameters.js

```diff
@@ -43,6 +47,10 @@ const TAB_VALUES = {
 const DynamicSimulationParameters = ({ user, hideParameters }) => {
     const classes = useStyles();
 
+    const dynamicSimulationUnavailability = useServiceUnavailabilty(
```

Comment:
```suggestion
    const isDynamicSimulationUnavailable = useServiceUnavailabilty(
```
rename all const to "is<Service>Unavailable" to improve code readability